### PR TITLE
 Translation Spanish [Part 1] // 1102 Words

### PR DESCRIPTION
**Pull request by Santu41**
Hi, I am a translator working for the Utopian.io community and I am interested participation in your project, doing the translation into Spanish. I hope we can work together..

the modified file was: ch20-06-graceful-shutdown-and-cleanup.md

### CODE REVIEW

File: second-edition/src/ch20-06-graceful-shutdown-and-cleanup.md [hunk 1]
```diff
@@ -36,13 +26,9 @@ impl Drop for ThreadPool {
 <span class="caption">Listing 20-22: Joining each thread when the thread pool
 goes out of scope</span>
 
-We loop through each of the thread pool `workers`, using `&mut` because `self`
-is itself a mutable reference and we also need to be able to mutate `worker`.
-We print out a message saying that this particular worker is shutting down, and
-then we call `join` on that worker's thread. If the call to `join` fails, we
-`unwrap` the error to panic and go into an ungraceful shutdown.
+Recorrimos cada uno de los `pool` del grupo de subprocesos, usando` & mut` porque `self` es en sí misma es una referencia mutable y también necesitamos poder mutar `worker`. Imprimimos un mensaje que dice que este worker en particular se está cerrando, y luego llamamos a `join` al hilo de ese worker. Si la llamada a `join` falla, nosotros `deploy` el error para entrar en pánico y realizar un apagado desagradable.
```
Comment (thecodix) [inline second-edition/src/ch20-06-graceful-shutdown-and-cleanup.md]: Would recommend a different approach for 'apagado desagradable', maybe 'cierre elegante', but is a personal opinion, wdyt?

File: second-edition/src/ch20-06-graceful-shutdown-and-cleanup.md [hunk 2]
```diff
@@ -1,23 +1,13 @@
 ## Graceful Shutdown and Cleanup
```
Comment (thecodix) [inline second-edition/src/ch20-06-graceful-shutdown-and-cleanup.md]: May consider translating title as well?

**Comment by thecodix** [comment]
Hey @Santu41 , thanks a lot for investing time on the translation, may I ask you to try and follow the format of the english version on all paragraphs and cut each line around the same width? (cant recall if its at 120 characters)

That way it will ease review/edition tasks no matter the screen used

Thanks a lot, and watch out for a couple minor comments as well :)

**Comment by thecodix** [comment]
Also, there is some conflicts with the main branch (due to a merge yesterday), do you think you can spend some minutes fixing them?

You need to merge the master branch of thecodix/book in your local branch again

https://help.github.com/articles/merging-an-upstream-repository-into-your-fork/

**Comment by Santu41** [comment]
the file inside my fork disappeared and I can not edit it, the "resolve conflicts" button appears deactivated as well

69dab4956a5dd7a114105e6f2c83d933

2213da5f4b062591e79dce1d59a91ee8

**Comment by thecodix** [comment]
You need to do it in your local machine, with an IDE or with git from command line

**Comment by thecodix** [comment]
I can assist you if you find issues trying to merge locally, lemme know

**Comment by Santu41** [comment]
It's my first time using the github platform, I'm not very familiar .. I would like ur help, I'm a translator, not a programmer haha

**Comment by thecodix** [comment]
No issues :)

Best will probably be, since its the first time you do changes, to:
1) copy the translated file in a local file in your computer
2) delete your local repo
3) make again a fork of thecodix/book (it will contain all updated files)
4) open a new PR for this same chapter, where you put the changes again

Apologises for the inconveniences!

**Comment by thecodix** [comment]
Unless you need it, I will close this PR as not needed anymore

**Comment by thecodix** [comment]
machine translated

**Pull request closed without merging**